carbon-design-system/design-language-website
# chore(project): update icon dependencies

Closes #1122

#### Changelog

**Changed**

- update to node 16
- update icons and pictogram packages to latest
- refactor icons usage to use new metadata format
- refactor icon usages to use `size` prop 
  - primarily the output of `npx @carbon/upgrade migrate icons-react-size-prop -w`
  - manually refactored the usages in `.mdx` files
- updated the gatsby theme to no longer use node-sass, https://github.com/carbon-design-system/design-language-website/pull/1123


#### Testing / Reviewing

- View the icons library, ensure no regressions
- View the pictograms library, ensure no regressions
- View the app icons library, ensure no regressions
- View other pages impacted by these changes (icon `size` usage changes), ensure no regressions
